src/normalizer/deunicode.rs

```diff
@@ -0,0 +1,107 @@
+use std::borrow::Cow;
+
+use deunicode::deunicode_char;
+
+use super::Normalizer;
+use crate::detection::{Language, Script};
+use crate::Token;
+
+/// Latin specialized [`Normalizer`] converting unicode chars into Ascii.
+///
+/// This Normalizer uses [`deunicode`] internally to normalize the provided token.
+pub struct DeunicodeNormalizer;
+
+impl Normalizer for DeunicodeNormalizer {
+    fn normalize<'o>(&self, mut token: Token<'o>) -> Box<dyn Iterator<Item = Token<'o>> + 'o> {
+        if !token.lemma().is_ascii() {
+            let mut char_map = Vec::new();
+            let mut lemma = String::new();
+            for c in token.lemma().chars() {
+                let deunicoded = deunicode_char(c).unwrap_or_default();
```

Comment:
What is a default `char`? [It looks like](https://doc.rust-lang.org/src/core/default.rs.html#188) the default `char` is `\x00` which is not good...